pkg/object/object.go

```diff
@@ -236,9 +295,85 @@ func (v *Volume) validate() error {
 	return nil
 }
 
+func (sd *SecretData) validate() error {
```

Comment:
Anyway the comment in stackoverflow itself explains, and I think that is what we want, we are not checking if the value is base64 encoded of something, we just wanna see if it base64 encoded? We don't care whose!